parts/linux/cloud-init/artifacts/components.json

```diff
@@ -202,18 +203,20 @@
       "downloadURL": "mcr.microsoft.com/oss/calico/node:*",
       "amd64OnlyVersions": [],
       "multiArchVersions": [
-        "v3.24.6"
+        "v3.24.6",
```

Comment:
should `calico/node` have a `prefetchOptimazations` section ?

Reply:
we haven't prefetched this thus far, though not sure if we really _should_ be

Reply:
I mean I don't actually want to push calcio so maybe the better thing to do is drop it completely. Also are we going to cache 4 versions when I bring in calcio 3.28?